docker-for-aws/iam-permissions.md

```diff
@@ -0,0 +1,313 @@
+---
+description: IAM permissions
+keywords: aws iam permissions
+title: Docker for AWS IAM permissions
+---
+
+Here is a list of IAM permissions that are required in order to use Docker for AWS.
+
+If you want to deploy Docker for AWS, your account will need to have these permission, or else the stack will not
```

Comment:
Before you deploy Docker for AWS, your account needs these permissions for the stack to deploy correctly If you create and use an IAM role with these permissions for creating the stack, CloudFormation will use the role's permissions instead of your own, using the AWS CloudFormation Service Role feature.